### DISCUSSION TITLE
Dynamic Recon Ops

### DISCUSSION THREAD

**Discussion by RickJamesBish** [Foxlider/FASTER#137] [Q&A] [answered]
How do I add Dynamic Recon Ops to the missions? For example I found he mission.pbo for DRO Tanoa. I copied it and placed it in the Missions and MPMissions folders. I don't know where else to put it so that FASTER sees it, or if FASTER will even see it.

**Comment by MildlyInterested** [comment, marked as answer]
Support is available on discord. https://discord.gg/2BUuZa3
